tests/gbs/test_sample.py

```diff
@@ -373,21 +375,94 @@ def test_output_sampled_modes(self):
 
 
 @pytest.mark.parametrize("dim", [4])
-def test_random_seed(dim, adj):
-    """Test for the function ``glassonion.sample.random_seed``. Checks that samples are identical
-    after repeated initialization of ``random_seed``."""
+def test_seed(dim, adj):
+    """Test for the function ``strawberryfields.gbs.sample.seed``. Checks that samples are identical
+    after repeated initialization of ``seed``."""
 
-    sample.random_seed(1968)
-    q_s_1 = sample.quantum_sampler(
-        A=adj, n_mean=2, samples=10, backend_options={"threshold": False}
-    )
-    u_s_1 = sample.uniform_sampler(modes=dim, sampled_modes=2, samples=10)
+    sample.seed(1968)
+    q_s_1 = sample.sample(A=adj, n_mean=2, samples=10, backend_options={"threshold": False})
+    u_s_1 = sample.uniform(modes=dim, sampled_modes=2, samples=10)
 
-    sample.random_seed(1968)
-    q_s_2 = sample.quantum_sampler(
-        A=adj, n_mean=2, samples=10, backend_options={"threshold": False}
-    )
-    u_s_2 = sample.uniform_sampler(modes=dim, sampled_modes=2, samples=10)
+    sample.seed(1968)
+    q_s_2 = sample.sample(A=adj, n_mean=2, samples=10, backend_options={"threshold": False})
+    u_s_2 = sample.uniform(modes=dim, sampled_modes=2, samples=10)
 
     assert np.array_equal(q_s_1, q_s_2)
     assert np.array_equal(u_s_1, u_s_2)
+
+
+quantum_samples = [
```

Comment:
The convention for constant values is to place them at the top of the file and use an all-uppercase name, like `QUANTUM_SAMPLES`. However, it seems like you only use these variables in the `TestToSubgraphs` tests. In this case I would recommend making them class-level constants.

Reply:
Ok, have done this! Thanks!